gensim/models/atmodel.py

```diff
@@ -928,6 +1046,23 @@ def get_author_topics(self, author_name, minimum_probability=None):
         Obtaining topic probabilities of each word, as in LDA (via `per_word_topics`),
         is not supported.
 
+        Parameters
+        ----------
+        author_name : str
+            Name of the author for which the topic distribution needs to be estimated.
+        minimum_probability : float, optional
+            Sets the minimum probability value for showing the topics of a given author.
+            
+        Returns
+        -------
+        list of 2-tuples
+            Topic distribution of an author as a list of topic ID and its probability.
+            
+        Example
+        -------
+        >>> author_vecs = [model.get_author_topics(author) for author in model.id2author.values()]
```

Comment:
This example doesn't work, you should define all variable & make all imports first.

Reply:
Simple test for example: I copy-paste it to REPL, run and this works successfully